apps/myaccount/src/protected-app.tsx

```diff
@@ -184,11 +186,23 @@ export const ProtectedApp: FunctionComponent<AppPropsInterface> = (): ReactEleme
             .catch((error: string) => {
                 throw new I18nInstanceInitException(error);
             });
+
+
     }, [ isAuthenticated ]);
 
+    // Extracting theme preference data using custom hook
```

Comment:
Add full stops at the end of comments. Please check other places as well.
```suggestion
    // Extracting theme preference data using custom hook.
```

Reply:
addressed with https://github.com/wso2/identity-apps/pull/6173/commits/651b7a87b8b4c871d6af2f41362a7f8759aa1121

Reply:
addressed with https://github.com/wso2/identity-apps/pull/6173/commits/651b7a87b8b4c871d6af2f41362a7f8759aa1121